### DISCUSSION TITLE
Flying above towns

### DISCUSSION THREAD

**Discussion by IkanTerbakor** [Vdauphin/HeartsAndMinds#1671] [Question and answer]
Hello!

How to reduce the speed that is needed when flying above town to prevent it from spawning stuff from 190 to about 100 kmh?

Cheers!

**Comment by Vdauphin** [comment]
Hello,

Change `190` to whatever you want
https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L101

Cheers
